cordova/build.sh

```diff
@@ -93,6 +93,16 @@ if [ ! -d $PROJECT ]; then
     checkOK
   fi
 
+  if [ $CURRENT_OS == "IOS" ]; then
+    cordova plugin add phonegap-plugin-push --variable SENDER_ID="XXXXXXX"
```

Comment:
For iOS you set the latest version of the plugin, but for Android your are using an specific version (1.2.3). Is that ok too?